test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs

```diff
@@ -131,6 +131,18 @@ public void SuppresssesInstrumentation()
             Assert.Equal(1, endCalledCount);
         }
 
+        [Fact]
+        public void EndpointConfigurationUsingEnvironmentVariable()
+        {
+            Environment.SetEnvironmentVariable("OTEL_EXPORTER_ZIPKIN_ENDPOINT", "http://fakeuri");
+
+            var options = new ZipkinExporterOptions();
+
+            Assert.Equal(new Uri(Environment.GetEnvironmentVariable("OTEL_EXPORTER_ZIPKIN_ENDPOINT")).AbsoluteUri, options.Endpoint.AbsoluteUri);
+
+            Environment.SetEnvironmentVariable("OTEL_EXPORTER_ZIPKIN_ENDPOINT", null);
+        }
```

Comment:
Need more tests to validate priority. For example if zipkin endpoint is set in code (https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/Console/TestZipkinExporter.cs#L45) *and* env variable -there must be a test which confirms the right one is picked.

Reply:
@cijothomas - Added that test now. Please review.

Reply:
Need to confirm which one takes priority. Will update after getting the confirmation.

Reply:
I think code takes higher priority than settings.

Reply:
I was thinking exact opposite ! Devs deploy code.. Ops can override things using ENV variable..

But i'd ask spec to clarify if Env variable takes higher priority or code takes..

Reply:
Keeping Incode precedence over environment variable as discussed offline. Please review Readme as well. Thanks!